samples/excel/44-shape/shape-textboxes.yaml

```diff
@@ -72,7 +72,10 @@ script:
 
         async function setup() {
             await Excel.run(async (context) => {
-                await OfficeHelpers.ExcelUtilities.forceCreateSheet(context.workbook, "Shapes");
+                context.workbook.worksheets.getItemOrNullObject("Shapes").delete();
+                const sheet = context.workbook.worksheets.add("Shapes");
+                await context.sync();
```

Comment:
Also, here and elsewhere, you probably don't need the "context.sync()" here (it's only causing an extra round-trip for absolutely no good reason).  The reason the original method was awaitable was because it *may* have needed to do a sync (but it only did it if it had to on older 2016 clients).  Whereas here you are doing it all the time.

Reply:
Before I go back through and test every Excel snippet to ensure this `context.sync()` is not necesssary, isn't it valid because of the delete beforehand? It ensures the workbook is in a ready state for what follows.

Reply:
It's just not necessary, making the code slower and adding confusion for no good reason.
For the latter, what I mean is that it calls into question what "context.sync()" even means.  Sure, performance aside, it doesn't hurt to stick it after _every_ API call -- but that defeats the point of batching (and makes perf be terrible).  So having extra syncs for no good reason hurts the conceptual integrity of explaining when you need syncs (which should be "as rarely as possible, only when needed")